api-catalog-ui/frontend/src/components/ExtraContents/BlogTile.test.jsx

```diff
@@ -27,28 +27,26 @@ describe('>>> BlogTile component tests', () => {
     });
     it('should render blog tile', () => {
         const blogTile = shallow(<BlogTile blogData={props.blogData} />);
-        expect(blogTile.find('[data-testid="blogs-image"]').exists()).toEqual(true);
         expect(blogTile.find('[data-testid="blog-title"]').first().prop('children')).toEqual('title');
         expect(blogTile.find('[data-testid="blog-description"]').exists()).toEqual(true);
-        expect(blogTile.find('[data-testid="author"]').first().prop('children')).toEqual('author');
-        expect(blogTile.find('[data-testid="pub-date"]').first().prop('children')).toEqual('Published: 123343');
+        expect(blogTile.find('[data-testid="blog-learn"]').exists()).toEqual(true);;
```

Comment:
double semi colon here, could make the linter fail too

Reply:
It is fine for tests. on other lines it also didn't fail

Reply:
Ok, I will add it into my todo